### ISSUE TITLE
issue with running of burrow

### ISSUE THREAD

**Issue by shub754503** [linkedin/Burrow#552]
can i run burrow without docker??

**Comment by chrnola** [comment]
Yes, Burrow is just a Go program. Check out the readme for more info on building, installing, and running: https://github.com/linkedin/Burrow#build-and-install
